EpiAware/src/EpiAware.jl

```diff
@@ -30,13 +30,11 @@ using Distributions,
     Parameters,
     QuadGK
 
+# Exported utilities
 export scan,
-    create_discrete_pmf,
-    growth_rate_to_reproductive_ratio,
-    generate_observation_kernel,
-    EpiModel,
-    log_daily_infections,
-    random_walk
+    create_discrete_pmf, growth_rate_to_reproductive_ratio, generate_observation_kernel
```

Comment:
why is scan on its own line but none of the others are? That aside splitting it out like this is nice

Reply:
The JuliaFormatter basic format standard is to let tuple arguments be flat until they hit a certain length and then change into column format.

Reply:
We can fiddle that if its annoying.